core/server/common/src/main/java/alluxio/master/journal/JournalInputStream.java

```diff
@@ -16,16 +16,16 @@
 import java.io.IOException;
 
 /**
- * This input stream retrieves {@link JournalEntry} from journal checkpoint files and journal log
- * files.
+ * This input stream retrieves {@link JournalEntry} from journal checkpoints and journal logs.
  */
 public interface JournalInputStream {
```

Comment:
- `AutoClosable`?
- Why we need both `JournalInputStream` and `JournalReader`? Can we have just one?